.github/workflows/release.yaml

```diff
@@ -32,3 +32,41 @@ jobs:
       - name: test-image
         run: |
           VERSION=3.6.99 ./scripts/test_images.sh
+      - name: save-image
+        run: |
+          docker image save -o /tmp/etcd-img.tar gcr.io/etcd-development/etcd
+      - name: upload-image
+        uses: actions/upload-artifact@v3
+        with:
+          name: etcd-img
+          path: /tmp/etcd-img.tar
+          retention-days: 1
```

Comment:
Why do we need to save & upload the image? Can we just add one more step `scan-image` in the same way as previous step `test-image`?

Reply:
Since we are using matrix to run parallel jobs, if we add a step, the entire `main` job with all the steps will be replicated for all the archs. 
In my understanding, the step `release` which might be resource-demanding will be replicated for all the archs, each time doing the exact same thing.

Reply:
Thanks for the clarification. 

Have you verified that `docker image save ...; docker load ...` can recover all images for different archs?

Reply:
Confirmed that it indeed loads all images for different archs.

```
Run docker load < /tmp/etcd-img.tar
  docker load < /tmp/etcd-img.tar
  shell: /usr/bin/bash -e {0}
Loaded image: gcr.io/etcd-development/etcd:v[3](https://github.com/etcd-io/etcd/actions/runs/8523801808/job/23347153138?pr=17679#step:4:3).6.99-ppc64le
Loaded image: gcr.io/etcd-development/etcd:v3.6.99-s390x
Loaded image: gcr.io/etcd-development/etcd:v3.6.99-amd6[4](https://github.com/etcd-io/etcd/actions/runs/8523801808/job/23347153138?pr=17679#step:4:5)
Loaded image: gcr.io/etcd-development/etcd:v3.6.99-arm64
```